internal/parse/parse.go

```diff
@@ -34,3 +36,52 @@ func Image(s string) error {
 
 	return nil
 }
+
+// DeploymentLogsForString parses the container logs of the specified ns/name
+// deployment, returning true if the string was found.
```

Comment:
The changes to this look good, could you add a TODO here, linking to #200, so that we can have a full bidirectional reminder? :)